google/cloud/security/inventory/pipelines/load_groups_pipeline.py

```diff
@@ -14,112 +14,109 @@
 
 """Pipeline to load GSuite Account Groups into Inventory."""
 
-from oauth2client.contrib.gce import AppAssertionCredentials
-from oauth2client.service_account import ServiceAccountCredentials
-
-from google.cloud.security.common.data_access import errors as data_errors
-from google.cloud.security.common.gcp_api import admin_directory
 from google.cloud.security.common.gcp_api import errors as api_errors
 from google.cloud.security.common.util import metadata_server
-from google.cloud.security.common.util.log_util import LogUtil
 from google.cloud.security.inventory import errors as inventory_errors
-from google.cloud.security.inventory import transform_util
-
+from google.cloud.security.inventory.pipelines import base_pipeline
 
-LOGGER = LogUtil.setup_logging(__name__)
-RESOURCE_NAME = 'groups'
-REQUIRED_SCOPES = frozenset([
-    'https://www.googleapis.com/auth/admin.directory.group.readonly'
-])
 
+class LoadGroupsPipeline(base_pipeline.BasePipeline):
+    """Pipeline to load groups data into Inventory."""
+    # TODO: Add unit tests.
 
-def _is_our_environment_gce():
-    """A simple function that returns a boolean if we're running in GCE."""
-    return metadata_server.can_reach_metadata_server()
+    RESOURCE_NAME = 'groups'
 
-def _can_inventory_google_groups(config):
-    """A simple function that validates required inputs for inventorying groups.
+    def __init__(self, cycle_timestamp, configs, admin_client, dao):
+        """Constructor for the data pipeline.
 
-    Args:
-        config: Dictionary of configurations built from our config.
+        Args:
+            cycle_timestamp: String of timestamp, formatted as YYYYMMDDTHHMMSSZ.
+            configs: Dictionary of configurations.
+            admin_client: Admin API client.
+            dao: Data access object.
 
-    Returns:
-        Boolean
-    """
-    required_gcp_execution_config = [
-        config.get('service_account_email'),
-        config.get('domain_super_admin_email')]
+        Returns:
+            None
+        """
+        super(LoadGroupsPipeline, self).__init__(
+            self.RESOURCE_NAME, cycle_timestamp, configs, admin_client, dao)
 
-    required_local_execution_config = [
-        config.get('service_account_email'),
-        config.get('service_account_credentials_file'),
-        config.get('domain_super_admin_email')]
+    def _can_inventory_google_groups(self):
+        """A simple function that validates required inputs to inventory groups.
 
-    if _is_our_environment_gce():
-        required_execution_config = required_gcp_execution_config
-    else:
-        required_execution_config = required_local_execution_config
+        Args:
```

Comment:
Nit: You don't need an Args section if there are no args. Same below where you've added an Args: None section.